x-pack/test/stack_functional_integration/apps/ccs/ccs_discover.js

```diff
@@ -130,11 +133,29 @@ export default ({ getService, getPageObjects }) => {
     });
 
     it('create index pattern for data from both clusters', async () => {
-      await PageObjects.settings.createIndexPattern('*:makelogs工程-*', '@timestamp', true, false);
+      await PageObjects.settings.createIndexPattern('*:makelogs工程-*', '@timestamp');
       const patternName = await PageObjects.settings.getIndexPageHeading();
       expect(patternName).to.be('*:makelogs工程-*');
     });
 
+    it('create local siem signals index pattern', async () => {
```

Comment:
As in this file it is already tested that an index pattern can be added, in order to improve execution time, as part of the data preparation we need for our test, we can probably add the index we need through the API.

Reply:
Such API is available since 7.11 but it's not publicly documented yet. See #99372.